app/src/atoms/MenuList/DropdownMenu.tsx

```diff
@@ -26,52 +26,100 @@ export interface DropdownMenuProps {
   filterOptions: DropdownOption[]
   onClick: (value: string) => void
   currentOption: DropdownOption
+  width?: string
+  dropdownType?: 'rounded' | 'neutral'
+  title?: string
 }
 
 // TODO: (smb: 4/15/22) refactor this to use html select for accessibility
 
 export function DropdownMenu(props: DropdownMenuProps): JSX.Element {
-  const { filterOptions, onClick, currentOption } = props
+  const {
+    filterOptions,
+    onClick,
+    currentOption,
+    width = '9.125rem',
+    dropdownType = 'rounded',
+    title,
+  } = props
   const [showDropdownMenu, setShowDropdownMenu] = React.useState<boolean>(false)
-  const toggleSetShowDropdownMenu = (): void =>
+  const toggleSetShowDropdownMenu = (): void => {
     setShowDropdownMenu(!showDropdownMenu)
+  }
   const dropDownMenuWrapperRef = useOnClickOutside<HTMLDivElement>({
-    onClickOutside: () => setShowDropdownMenu(false),
+    onClickOutside: () => {
+      setShowDropdownMenu(false)
+    },
   })
 
+  const DROPDOWN_STYLE = css`
+    flex-direction: ${DIRECTION_ROW};
+    background-color: ${COLORS.white};
+    cursor: pointer;
+    padding: ${SPACING.spacing8} ${SPACING.spacing12};
+    border: 1px ${BORDERS.styleSolid}
+      ${showDropdownMenu ? COLORS.blue50 : COLORS.grey50};
+    border-radius: ${dropdownType === 'rounded'
+      ? BORDERS.borderRadiusFull
+      : BORDERS.borderRadius4};
+    align-items: ${ALIGN_CENTER};
+    justify-content: ${JUSTIFY_SPACE_BETWEEN};
+    width: ${width};
+
+    &:hover {
+      border: 1px ${BORDERS.styleSolid}
+        ${showDropdownMenu ? COLORS.blue50 : COLORS.grey55};
+    }
+
+    &:active {
+      border: 1px ${BORDERS.styleSolid} ${COLORS.blue50};
+    }
+
+    &:focus-visible {
+      border: 1px ${BORDERS.styleSolid} ${COLORS.grey55};
+      outline: 2px ${BORDERS.styleSolid} ${COLORS.blue50};
+      outline-offset: 2px;
+    }
+
+    &:disabled {
+      background-color: ${COLORS.transparent};
+      color: ${COLORS.grey40};
+    }
+  `
+
   return (
-    <>
+    <Flex flexDirection={DIRECTION_COLUMN}>
+      {title !== undefined ? (
```

Comment:
From the props type, `title?: string | null` so I'd recommend the following

```suggestion
      {title != null ? (
```